### DISCUSSION TITLE
How to merge Renovate pull requests while enabling the checksum verification

### DISCUSSION THREAD

**Discussion by sheldonhull** [aquaproj/aqua#2464] [Deprecated (Question and Answer)] [answered]
### Question

I use renovate and aqua. 
Since I use the check sum verification being required, I've noticed that renovate doesn't like this. It stops the auto merge from occurring because the pool request was modified externally from the renovate process.

How do you get around this?
I do like force to check some policies. It feels like a nice way to have better supply chain security, but I also don't want to break renovate from functioning correctly.

Please note that I am working in a mixed environment of GitHub plus as your op so purely GitHub actions won't help me. 

### Example Code

```console
$ 
```

Configuration

```yaml

```

### Reference

*

**Comment by suzuki-shunsuke** [comment]
Hmm. I'm not sure if I understand your situation correctly, but on https://github.com/aquaproj/aqua we enable the checksum verification and merge pull requests by Renovate automatically.

- https://github.com/aquaproj/aqua/pull/2438

We merge pull requests by enabling [GitHub auto-merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request).

**Reply by suzuki-shunsuke** [reply, marked as answer]
If auto-merge gets disabled because `Base branch was modified`, maybe re-enabling auto-merge by GitHub Actions is helpful.

https://github.com/suzuki-shunsuke/reenable-automerge-action

**Reply by sheldonhull** [reply]
Got you. So renovate won't automerge any longer, so I'd need to use github actions or the action you created. For azure devops, it's most noticeable when I don't get to the update for a bit. Renovate stops updating the pending pull request due the change by the renovate checksum update, which means it gets stale.

I'll consider this answered though as that's clear enough for my needs and will have to adjust my workflow.
